ivr/src/test/java/org/motechproject/ivr/it/OutboundCallServiceBundleIT.java

```diff
@@ -139,4 +143,68 @@ public void shouldExecuteJsonRequest() throws IOException {
         assertEquals(1, callDetailRecords.size());
         assertEquals("conf789", callDetailRecords.get(0).getConfigName());
     }
+
+    @Test
+    public void shouldCreateFileAndWriteParams() throws IOException {
+        Path path = Files.createTempFile("temp", ".tmp");
+        path.toFile().deleteOnExit();
+
+        String uri = path.toUri().toString();
+
+        Config config = new Config("conf159", false, null, null, null, null, null, null, HttpMethod.GET, false, uri, false, null);
+
+        configService.updateConfigs(Arrays.asList(config));
+
+        Map<String, String> params = new HashMap<>();
+        params.put("api_key", "qwerty123");
+        params.put("message_id", "123123");
+        params.put("channel", "ivr");
+        params.put("status_callback_url", "http://someUrl.com");
+        params.put("subscribers", "[{\"phone\":\"48700123123\",\"language\":null}]");
+
+        List<String> resultList = new ArrayList<>();
+        for(Map.Entry<String, String> entry: params.entrySet()){
+            resultList.add(String.format("%s: %s", entry.getKey(), entry.getValue()));
+        }
+
+        outboundCallService.initiateCall(config.getName(), params);
+
+        List<String> fileOutput = Files.readAllLines(path);
+
+        assertTrue(fileOutput.containsAll(resultList));
+    }
+
+    @Test
+    public void shouldCreateFileWithParamName() throws IOException, URISyntaxException {
+        Path path = Files.createTempFile("temp", ".tmp");
+        path.toFile().deleteOnExit();
+
+        String fileName = path.getFileName().toString();
+
+        String uri = path.toUri().toString().replace(fileName, "[fileName]");
+
+        Config config = new Config("conf159", false, null, null, null, null, null, null, HttpMethod.GET, false, uri, false, null);
+
+        configService.updateConfigs(Arrays.asList(config));
+
+        Map<String, String> params = new HashMap<>();
+        params.put("api_key", "qwerty123");
+        params.put("message_id", "123123");
+        params.put("channel", "ivr");
+        params.put("status_callback_url", "http://someUrl.com");
+        params.put("subscribers", "[{\"phone\":\"48700123123\",\"language\":null}]");
+
+        List<String> result = new ArrayList<>();
```

Comment:
This is pretty much the same as the previous test. Why not just have this one test?